### ISSUE TITLE
Add reccurring notification logic

### ISSUE THREAD

**Issue by Seeru-crypto** [Seeru-crypto/Dater_BE#10]
Add logic for recurring notifications. 
so that user can select either: no recurring notifications, daily or every other day notifications. 
The selected options will be stored with the Event data.

**Comment by JPunnar** [comment]
“Every year” already exists?

**Comment by Seeru-crypto** [comment]
> “Every year” already exists?

yes, annual notifications exist, but once a event has triggered a reminder, there will be x time before the actual date of the event. In that time the user can select to recieve additional notification for example: The user will recieve the notification once, every other day, every day for a given event.
